src/components/general/header/Header.tsx

```diff
@@ -26,13 +55,26 @@ const Header = () => {
                                 />
                             </Link>
                         </Col>
-                        <Col className="d-flex flex-column justify-content-center">
-                            <h2 className="w-auto fs-3 fs-lg-2 tc-primary fw-bold">
-                                Marketplace Prototype
-                            </h2>
-                            <h3 className="fs-4 fs-lg-3 tc-grey fw-lightBold">
-                                The Taxonomic Expertise and Services Marketplace
-                            </h3>
+                        <Col>
+                            <Row>
+                                <Col xs={{ span: 12 }} lg="auto"
+                                    className="d-flex flex-column justify-content-center"
+                                >
+                                    <h2 className="w-auto fs-3 fs-lg-2 tc-primary fw-bold">
```

Comment:
These are all custom classes I set in the app.css file. They use the same approach as the Bootstrap classes.
So:
- tc-primary: tc is short for text-color, primary indicates it should use the primary color, so primary colored text.
- fs-3: fs is short for font-size, 3 means to use font size 3, there is a range from font size 1 till 5. 3 is around 14px, the default being 12px.
- fs-lg-2: again fs is font-size, 2 means to use font-size 2 and lg means to use it for large screens like laptops or other monitors. Defining such a property overrides the fs-3 since it is specific to a certain screen width. So the rule of thumb is: use classes without any screen width indication or xs (extra small) first, then look for other classes that might be applied to large screens. This all depends upon the type of device the user is using.